bluesky_kafka/tests/conftest.py

```diff
@@ -4,6 +4,22 @@
 import ophyd.sim
 
 
-@pytest.fixture(scope='function')
+# get bootstrap server IP from command line
+def pytest_addoption(parser):
```

Comment:
This is neat! :+1:

Reply:
But it caused the "I need to cd kafka_bluesky" problem.